sympy/functions/special/beta_functions.py

```diff
@@ -120,3 +121,83 @@ def _eval_conjugate(self):
 
     def _eval_rewrite_as_gamma(self, x, y, **kwargs):
         return self._eval_expand_func(**kwargs)
+
+
+###############################################################################
+########################### INCOMPLETE BETA  FUNCTION #########################
+###############################################################################
+
+
+
+class betainc(Function):
+    r"""
+    The incomplete beta function is a generalization of the beta function,
+
+    .. math::
+        \mathrm{B}_{z}(a,b) := \int^{z}_{0} t^{a-1} (1-t)^{b-1} \mathrm{d}t.
+
+    Incomplete beta function satisfies properties like:
+
+    .. math::
+        \mathrm{B}_{1}(a,b) = \mathrm{B}(a,b) \\
+
+    See Also
+    ========
+    Gamma function, Upper incomplete gamma function, Lower incomplete gamma function,
+    Polygamma function, Log Gamma function, Digamma function, Trigamma function.
+
+    References
+    ==========
+
+    .. [1] https://en.wikipedia.org/wiki/Beta_function#Incomplete_beta_function
+    .. [2] http://mathworld.wolfram.com/IncompleteBetaFunction.html
+    """
+    nargs = 3
```

Comment:
`nargs` should be a Set:
```
In [1]: sin.nargs                                                                                                                              
Out[1]: {1}

In [2]: type(_)                                                                                                                                
Out[2]: sympy.sets.sets.FiniteSet
```

Reply:
ValueError: FiniteSet(2) is not an integer

Reply:
I see now that it is not necessary to declare `nargs` at all. It will be inferred from the `eval` class method by the metaclass for `Function`. I think that you can just remove this line and then nargs will automatically become `FiniteSet(3)`.